src/libs/libjpeg_turbo/lv_libjpeg_turbo.c

```diff
@@ -369,8 +393,10 @@ static lv_draw_buf_t * decode_jpeg_file(const char * filename)
             jpeg_read_scanlines(&cinfo, buffer, 1);
 
             /* Assume put_scanline_someplace wants a pointer and sample count. */
-            lv_memcpy(cur_pos, buffer[0], stride);
-            cur_pos += decoded->header.stride;
+            dispose_buffer(cur_pos, buffer[0], line_index, cinfo.output_width, cinfo.output_height, decoded->header.stride,
```

Comment:
Can you please rename function to _rotate_buffer()_? 

_Dispose_ means to me releasing a resource, cleaning up, for example in .NET we use for that purpose.
But there is also a [dispose pattern](https://en.wikipedia.org/wiki/Dispose_pattern) for releasing resources, so dispose is not the best name for rotating image parts.